schema/bancor/view_smart_tokens.sql

```diff
@@ -0,0 +1,271 @@
+CREATE OR REPLACE VIEW bancor.view_smart_tokens (symbol, contract_address, decimals) AS VALUES
```

Comment:
This should ideally match the order of `erc20.tokens` i.e. (contract_address, symbol, decimals)

Reply:
Sorry for that one, tricky to debug when can't really test the views. :) 
Committed order fix.